src/ImpactX.cpp

```diff
@@ -65,8 +65,25 @@ namespace impactx
     void ImpactX::MakeNewLevelFromScratch (int lev, amrex::Real time, const amrex::BoxArray& ba,
                                           const amrex::DistributionMapping& dm)
     {
-        // todo data_mf.define(ba, dm, 1, 0);
-        amrex::ignore_unused(lev, time, ba, dm);
+        amrex::ignore_unused(time, ba, dm);
+
+        // set human-readable tag for each MultiFab
+        auto const tag = [lev]( std::string tagname ) {
+            tagname.append("[l=").append(std::to_string(lev)).append("]");
+            return amrex::MFInfo().SetTag(std::move(tagname));
+        };
+
+        // charge (rho) mesh
+        amrex::BoxArray cba = ba;
+        // for MR levels (TODO):
+        //cba.coarsen(refRatio(lev - 1));
+
+        auto const rho_nodal_flag = amrex::IntVect::TheNodeVector();
```

Comment:
Reviewer: double check what we want for now as discretization:
[nodal or cell-centered](https://amrex-codes.github.io/amrex/docs_html/Basics.html#box)

Reply:
In order to re-use WarpX/ablastr code, we need to have it nodal for now.